sdk/servicebus/azure-servicebus/azure/servicebus/_common/constants.py

```diff
@@ -61,6 +61,7 @@
 CONSUMER_IDENTIFIER = VENDOR + b":receiver-name"
 UAMQP_LIBRARY = "uamqp"
 PYAMQP_LIBRARY = "pyamqp"
+OPERATION_TIMEOUT = VENDOR + b":timeout"
```

Comment:
maybe this? to make it clear that these reference the same thing/the constant is already defined. But, this naming makes it clear that it's used as a config, and not in an error scenario.
```suggestion
# operation timeout used by client to reset server timeout value
OPERATION_TIMEOUT = ERROR_CODE_TIMEOUT
```
or should this be deleted and ERROR_CODE_TIMEOUT used everywhere?